example-setup/scripts/install_generic.sh

```diff
@@ -31,10 +33,34 @@ cat <<EOF> /etc/consul.d/consul.json
   "data_dir": "/tmp/consul",
   "bootstrap_expect": 3,
   "server": true,
-  "enable_script_checks": true
+  "enable_script_checks": true,
+  "disable_remote_exec": true,
+  "start_join": ["192.168.2.11", "192.168.2.12", "192.168.2.13"]
 }
 EOF
 
+cat <<EOF> /etc/consul.d/aclmaster.json
+{
+  "acl_datacenter": "cbk1",
+  "acl_default_policy": "allow",
+  "acl_down_policy": "allow",
```

Comment:
Shouldn't we deny any access operate in a whitelist mode? 
I think of something like this: https://www.consul.io/docs/guides/acl.html#bootstrapping-acls